src/hooks/useModal.tsx

```
@@ -0,0 +1,36 @@
+import React, { useContext, createContext, useState, ElementType } from 'react'
+
+type ModalsType = {
+  Component: ElementType
+  props?: any
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)